W-14069450-botsessioninformation-se (#293)
mulesoft/docs-rpa: rpa-bot/1.3/modules/ROOT/pages/configuration.adoc

[[common-settings]]
== Common Settings

> image:bot-configurator-common.png[The RPA Bot Configurator application showing the Common settings panel, 50%, 50%]

Reviewer: check other alt text for this change

* *RPA Bot ID*
+
ID generated during the installation process of RPA Bot, which identifies the bot in RPA Manager (even if the IP or MAC-Address changes). If you need a new ID, uninstall and then reinstall RPA Bot.
* *RPA Manager to connect*
+
The URL address of the RPA Manager instance the bot connects to. There is no default setting.
* *API Key*
+
The API key entered during the installation of the robot. You can change this setting.
* *TEST CONNECTION*
+
Click *TEST CONNECTION* to verify that the bot can establish a connection to RPA Manager.

> * *Alive interval*

Reviewer: suggest to the UI designers to make these items init cap (or at least consistent) :)

Author: @tasmoore Please check this for the next Bot (Configurator) release. There are more of these.... Thank you

Author: included in bug mentioned somewhere in this conversation

+
Time interval in milliseconds after which the bot connects to the RPA Manager to show that it is still alive.
* *Check interval*
+
Time interval in milliseconds after which the bot connects to RPA Manager and checks for provided packages to collect (such as scheduling tasks or activities).

> * *Cleanup activities on start*

Reviewer: Suggest to the UI designer to use "Clean up" (verb form) :)

Author: There is a bug for this already 
@tasmoore [(Bot) Bot Configurator - UI CX/UX Bugs](https://gus.lightning.force.com/a07EE00001cJLQOYA4)

+
Select this option to delete the activities that are no longer assigned to a configuration but are still stored locally in the bot.
* *Cleanup runtime packages on next start*
+
Select this option to delete all runtime packages that are stored locally in the bot. The process activities retrieve the required runtime packages from the RPA Manager on the next run. After the runtime packages are deleted, the Bot Configurator deselects this setting.
* *Maximum number of sessions*
+

> Set the maximum number of Secure Sessions that the bot can use. Enabling this setting, limits but not overwrites the same setting in RPA Manager Bot Management.

Reviewer: ```suggestion
Set the maximum number of secure sessions that the bot can use. Enabling this setting limits, but doesn't overwrite, the same setting in RPA Manager Bot Management.
```

Reviewer: What does "limits... the same setting" mean? Does it limit the number of secure sessions but doesn't overwrite the setting elsewhere?

Author: Yes
Example
Bot Setting + Manager Setting = Valid Setting
42 + 1 = 1
1 + 42 = 1

Reviewer: Can you revise to make clear?

Enabling this setting limits the number of secure sessions, but doesn't overwrite the same setting in RPA Manager Bot Management.

[[session-settings]]
== Session Settings

> If you want to have a default user account, that the RPA Bot uses to log in to the session on the computer, where the bot runs processes, enter the credentials here. When users create a process run configuration in RPA Manager, they can choose this user or specify another there.

Reviewer: ```suggestion
To specify a default user account that the RPA Bot uses to log in to the session and run processes on the computer, enter the credentials here. When users create a process-run configuration in RPA Manager, they can choose this user or specify another there.
```

> image:bot-configurator-session-settings.png[The RPA Bot configurator application showing the Secure Session settings panel, 50%, 50%]

Reviewer: ```suggestion
image:bot-configurator-session-settings.png[The RPA Bot Configurator application showing the Secure Session settings panel, 50%, 50%]
```

Reviewer: (to match other mentions)

* *Default Session User*
+

> If you want the bot to have a default user to log in to the session for the execution of a process, enter the credentials here. If the environment of the computer the bot is installed on is embedded in a domain and you want to use a local user, write the user name as follows: `.\username`. For domain users, use the following form: `mydomain\username`.

Reviewer: Can you use similar terminology as on line 64?

To specify that the RPA Bot use a default user account to log in to the session and run processes on the computer, enter the credentials here.

* *Send images for process streaming*
+

> Enable this checkbox to transfer screenshots used for process streaming to RPA Manager. Disable the checkbox if not everyone on the process team is allowed to view the processed data. For the Process Streaming to work, the usage of the Windows Taskbar must be enabled in the process run configuration, too.

Reviewer: We use select and deselect for [checkboxes](https://confluence.internal.salesforce.com/display/MTDT/MuleSoft+CX+Writing+Style+Reference#MuleSoftCXWritingStyleReference-%22select,%22%22choose%22).

[[secure-session]]
== Secure Session Settings

> image:bot-configurator-secure-session.png[The RPA Bot configurator application showing the Secure Session Settings panel, 50%, 50%]

Reviewer: init cap?

== See Also

* xref:rpa-manager::botmanagement-overview.adoc#getting-started[RPA Manager - Bot Management]

> * xref:rpa-manager::processautomation-deploy.adoc[RPA Manager - Deploying Automations]

Reviewer: Do you need the hard-coded link text?

Author: Added this to the Docs Redesign project
